docs/hub/models-adding-libraries.md

```diff
@@ -184,7 +184,7 @@ All third-party libraries are Dockerized, so you can install the dependencies yo
 
 ### Register your libraries supported tasks on the hub
 
-To register the tasks supported by your library on the hub you'll need to add a mapping from your library name to its supported tasks in this [file](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/library-to-tasks.ts). This will ensure the inference API is registered for tasks supported by your model. This file is automatically generated as part of a [GitHub Action](https://github.com/huggingface/api-inference-community/actions/workflows/python-api-export-tasks.yaml) in the [
+To register the tasks supported by your library on the hub you'll need to add a mapping from your library name to its supported tasks in [library-to-tasks.ts file](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/library-to-tasks.ts). This will ensure the inference API is registered for tasks supported by your model. This file is automatically generated as part of a [GitHub Action](https://github.com/huggingface/api-inference-community/actions/workflows/python-api-export-tasks.yaml) in the [
```

Comment:
I find this a bit confusing. It says you have to edit the file and then that it's automatically generated. The GitHub action generates [something similar](https://github.com/huggingface/api-inference-community/actions/runs/7017787981/job/19091817114), to the [`.ts` file](https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/library-to-tasks.ts); for example, `audio-source-separation` is commented in the `.ts` file but not in the generated result.

My understanding is that the GitHub action provides the source of truth to the inference API, but I don't fully understand the workflow or relationship with `library-to-tasks.ts`.